Packs/ContentManagement/Scripts/CustomPackInstaller/CustomPackInstaller_test.py

```diff
@@ -57,6 +57,23 @@ def test_install_custom_pack_success(mocker, pack_id, context, err_massage, res)
     assert error_message == err_massage
 
 
+@pytest.mark.parametrize(
```

Comment:
Looks good, let's add more test cases:
1. A case where only Pack.zip is received (to make sure we still support that)
2. A case where no pack id is matched to ensure an error message is received on that case.
Also, please add a given-when-then documentation to give more information about what each case is testing and what is expected.

Reply:
@YuvHayun , I simplified the tests by adding to the existing success test, and added given-when-then comments (I'm not really sure how to do those so let me know if there's a better way). Also, that existing test would take care of #1. Regarding #2, there are also existing tests for install failures, but I'm not sure how to specifically mock no matching, since it looks like the existing tests just specify '' or 'OK' manually.

Reply:
I edit the given-when-then docs a bit and pushed the changes.